modules/auth/repo_form.go

```diff
@@ -365,7 +365,7 @@ func (f *MergePullRequestForm) Validate(ctx *macaron.Context, errs binding.Error
 
 // NewReleaseForm form for creating release
 type NewReleaseForm struct {
-	TagName    string `binding:"Required"`
+	TagName    string `binding:"Required;GitRefName"`
```

Comment:
but https://github.com/go-gitea/gitea/blob/master/templates/repo/release/new.tmpl#L20 the form name is `tag_name`.